### ISSUE TITLE
is still relevant with current version of razorpay?

### ISSUE THREAD

**Issue by vaibhavmule** [ayushj31/django-razorpay#2]
I haven't cloned this project to check, but this is a just quick question.

**Comment by priyadhoundiyal** [comment]
I wanted to use this too, but there has been no activity on the repo since 3 years and there's no documentation either. I've been using the [Razorpay Python Client](https://github.com/razorpay/razorpay-python) which is fairly straightforward to use, if you're looking for an alternative :)

**Comment by vaibhavmule** [comment]
Yes, this is an implementation of `razorpay` library which you mentioned. 

Also, API keys are [committed](https://github.com/ayushj31/django-razorpay/blob/e0c88d0b7d708bccad6bbfaa39d05a9a345b0908/payment_gateway/razorpay_gateway/views.py#L8) in this repo.
